tee-supplicant/src/tee_supplicant.c

```
@@ -821,6 +880,15 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	if (daemonize) {
```

We should perhaps call `plugin_load_all()` before this so we can return that error to the parent too.

---

OK